docs/publish/publish-office-add-ins-to-appsource.md

```diff
@@ -315,37 +334,53 @@ Replace the placeholders as follows:
 The following is an example.
 
 ```
+// Only the current user
 [HKEY_CURRENT_USER\Software\Microsoft\Office\16.0\Wef\AutoInstallAddins\Word\ContosoAdd-in] 
 "AssetIds"="WA999999999"
+
+// All users of the computer
+[HKEY_LOCAL_MACHINE\Software\Microsoft\Office\16.0\AutoInstallAddins\Word\ContosoAdd-in] 
+"AssetIds"="WA999999999"
 ```
 
- The exact code will depend on your installation framework and programming language. The following is an example in C#.
+The exact code will depend on your installation framework and programming language. The following is an example in C#.
 
- ```csharp
+> [!NOTE]
+> To install the add-in for all users, change this code so that `WriteRegisterKeys` takes a bool parameter. The method should set the `supportLocalMachine` variable to the value that is passed: `true` to install for all users, `false` to install for only the current user.
```

Comment:
I think this should either be `bool` or Boolean 
```suggestion
> To install the add-in for all users, change this code so that `WriteRegisterKeys` takes a `bool` parameter. The method should set the `supportLocalMachine` variable to the value that is passed: `true` to install for all users, `false` to install for only the current user.
```